api/middleware/inbound.go

```diff
@@ -71,6 +75,13 @@ func (h unaryHandlerWithMiddleware) Handle(ctx context.Context, req *transport.R
 	return h.i.Handle(ctx, req, resw, h.h)
 }
 
+func (h unaryHandlerWithMiddleware) Introspect() *introspection.Handler {
```

Comment:
It actually is. Because generic idl introspection must be forwarded from handlers, trough middlewares, up to the table router.